examples/src/examples/graphics/multi-draw-instanced-multi-platform.example.mjs

```diff
@@ -0,0 +1,231 @@
+import files from 'examples/files';
+import { rootPath, deviceType } from 'examples/utils';
+import * as pc from 'playcanvas';
+
+// @config DESCRIPTION Multi-draw instanced rendering of multiple primitives in a single call.
```

Comment:
Please expand this a bit - explain the need for workaround / some light details.